NativeYoutube.xcodeproj/project.pbxproj

```diff
@@ -506,7 +510,7 @@
 				CURRENT_PROJECT_VERSION = 10;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"NativeYoutube/Preview Content\"";
-				DEVELOPMENT_TEAM = 4538W4A79B;
```

Comment:
No Need to push your DEV ID

Reply:
😅 I definitely need to find a better work-flow so we don't constantly override this file with our developer id.

Reply:
To be honest i can't avoid pushing this file (as it also means not pushing files additions/deletions), and i don't really know if there is a way in XCode to avoid pushing the dev id :0